gateway/1.4/modules/ROOT/pages/_partials/task-config-liveness-probe.adoc

```diff
@@ -0,0 +1,99 @@
+//tag::intro1[]
+
+Flex Gateway offers the following CLI command to test the health of a Flex Gateway:
+
+[source,ssh]
+----
+flexctl probe --check=liveness
+----
+
+This command returns either a status code of `0` for a healthy Flex Gateway or `1` for an unhealthy Flex Gateway.
+
+//end::intro1[]
+
+//tag::intro2[]
+By using the liveness check command, you can either configure a Kubernetes liveness probe or a Docker health check command to restart a failed Flex Gateway.
+
+For Flex Gateways running on Linux, you can still run the liveness check command. However, there is not an easy to automate the Linux machine to restart when the command returns an unhealthy status.
```

Comment:
IMHO, I would try to avoid saying that theres not an easy way to automate the restart of the process. There might be expert Linux users reading this that know how to do it. Also, I suggest replacing "the command returns an unhealthy status" with "when the check/probe fails"

```suggestion
For Flex Gateways running on Linux, you can still run the liveness check command manually. However, we don't provide an automated way to restart the process when the check fails.
```